### PR TITLE
feat: add tokens and component classes

### PR DESCRIPTION
**Pull request by AnnaRybkina**
This is an attempt to join our 3 repos in one.
What is added:
https://github.com/warp-ds/component-classes
https://github.com/warp-ds/tokens

Result after build now:
image

**TODO:**

- [x] update README:s
- [x] we will now start publish to npm, double check if everything is added
- [ ] update with latest changes in tokens and component-classes - will do that as a last step

### CODE REVIEW

**Comment by BalbinaK** [comment]
Nice! I agree that keeping token-related files under `/tokens` path makes sense. 
Would we remove files like README and CHANGELOG from folders and keep just one for all on the root level?

**Comment by AnnaRybkina** [comment]
> Nice! I agree that keeping token-related files under `/tokens` path makes sense. Would we remove files like README and CHANGELOG from folders and keep just one for all on the root level?

Yea, I think so. Let's clean them up. I'll remove CHANGELOG:s and update the root README with everything we have under local README:s and then I'll delete them too

File: package.json [hunk 1]
```diff
@@ -3,21 +3,45 @@
...
+    "./component-classes/classes": "./component-classes/classes.js",
+    "./component-classes/shortcuts": "./component-classes/shortcuts.js"
+  },
+  "types": "./component-classes/index.d.ts",
+  "files": [
+    "./component-classes/index.js",
+    "./component-classes/classes.js",
+    "./component-classes/shortcuts.js",
+    "./component-classes/index.d.ts"
+  ],
```
Comment (BalbinaK) [inline package.json:26]: In [component-classes repo](https://github.com/warp-ds/component-classes/blob/alpha/package.json#L16) we also had `dist/assets` folder here. Just checking if it's no longer something we need to publish to npm.

File: README.md [hunk 2]
```diff
@@ -16,6 +17,70 @@ or
...
+
+#### Usage of tokens
+
+If you have new components to add tokens to (if you only want to update existing tokens proceed to step 2):
+
+1. Under applicable brand create a new yml file and preferably name it the same as the new component(e.g. button.yml if it is a button component)
+2. Update tokens needed.
+3. Push your changes.
+4. The CSS files for each brand will be built and published to Eik. Please find your release [here](https://github.com/warp-ds/css/actions/workflows/release.yml) and search for the css urls which were published under `Eik login and publish` step.
+! Be aware that it takes some time for Eik to update the aliased version so if you need to test your changes immediately use the full version and not an alias.
```
[the remark this answers is not in the document]
Comment (AnnaRybkina) [reply README.md:81]: https://github.com/warp-ds/css/pull/6/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R32 not after this PR is merged

[the remark this answers is not in the document]
Comment (AnnaRybkina) [reply README.md:81]: yes, good catch. I'll update

File: package.json (hunk 1 above)
Comment (AnnaRybkina) [reply package.json:26]: Yea, I figured that we probably do not need to add it to npm as the EIK version of the css is supposed to be used.
I can't even see it here https://www.npmjs.com/package/@warp-ds/component-classes?activeTab=code 🤔 Maybe it never worked. Or maybe I'm missing something here 😅

Comment (BalbinaK) [reply package.json:26]: Sounds right!

**Review by BalbinaK — APPROVED** [review]
Looks good to me!

**Pull request merged**